package.json

```diff
@@ -92,6 +93,7 @@
     "apollo-link": "1.2.11",
     "apollo-link-http": "1.5.14",
     "apollo-link-persisted-queries": "0.2.2",
+    "azure-functions-ts-essentials": "^1.3.2",
```

Comment:
Same missing `package-lock.json` update for this package.

Reply:
[same](https://github.com/apollographql/apollo-server/pull/2487#discussion_r268906353)